### PR TITLE
STONEBLD-1208: provide more info to proposed pac merge request

### PR DESCRIPTION
**Pull request by tkdchen**

### CODE REVIEW

**Comment by tkdchen** [comment]
Example: https://github.com/cqi-stonesoup-test/devfile-sample-python-basic/pull/3

File: controllers/component_build_controller_pac.go [hunk 1]
```diff
@@ -72,6 +72,17 @@ const (
...
+var mergeRequestDescription = strings.ReplaceAll(
+	`# Pipelines as Code configuration proposal
+
+To start the PipelineRun, add a new comment with content ~/ok-to-test~
+
+For more detailed information about running a PipelineRun, please refer to Pipelines as Code documentation [Running the PipelineRun](https://pipelinesascode.com/docs/guide/running/)
+
+To customize the proposed PipelineRuns after merge, please refer to [Build Pipeline customization](https://redhat-appstudio.github.io/docs.stonesoup.io/Documentation/main/getting-started/build_service.html)
+`,
+	"~", "`")
```
Comment (psturc) [inline controllers/component_build_controller_pac.go:84]: would it make sense to rather added it to const?
```suggestion
const mergeRequestDescription = `# Pipelines as Code configuration proposal

To start the PipelineRun, add a new comment with content ` + "`/ok-to-test`" + `

For more detailed information about running a PipelineRun, please refer to [Pipelines as Code documentation](https://pipelinesascode.com/docs/guide/running/)

To customize the proposed PipelineRuns after merge, please refer to [Build Pipeline customization](https://redhat-appstudio.github.io/docs.stonesoup.io/Documentation/main/getting-started/build_service.html)`
```

**Review by psturc — COMMENTED** [review]
just one comment, otherwise lgtm

File: controllers/component_build_controller_pac.go (hunk 1 above)
Comment (tkdchen) [reply controllers/component_build_controller_pac.go]: Updated

**Review by psturc — APPROVED** [review]
[result](https://github.com/redhat-appstudio-qe/devfile-sample-hello-world/pull/8151#issue-1701605346)

/lgtm

**Review by Michkov — APPROVED** [review]

**Pull request merged**